mmdet/apis/ote/extension/datasets/mmdataset.py

```diff
@@ -56,17 +61,24 @@ def get_annotation_mmdet_format(dataset_item: DatasetItemEntity, labels: List[La
         n = len(class_indices)
         gt_bboxes.extend([[box.x1 * width, box.y1 * height, box.x2 * width, box.y2 * height] for _ in range(n)])
         gt_labels.extend(class_indices)
+        if hasattr(annotation.shape, "points"):
+            points = []
+            for p in annotation.shape.points:
+                points.extend([p.x * width, p.y * height])
+            assert len(points) % 2 == 0
+            gt_polygons.append([np.array(points)])
 
     if len(gt_bboxes) > 0:
         ann_info = dict(
             bboxes=np.array(gt_bboxes, dtype=np.float32).reshape(-1, 4),
             labels=np.array(gt_labels, dtype=int),
-        )
+            masks=PolygonMasks(
+                gt_polygons, height=height, width=width) if gt_polygons else [])
     else:
         ann_info = dict(
             bboxes=np.zeros((0, 4), dtype=np.float32),
             labels=np.array([], dtype=int),
-        )
+            masks=[])
```

Comment:
Is it Ok, to have a list, but no numpy.ndarray here?

Reply:
either an empty list or an empty NumPy array would be fine here